cli/src/funTest/kotlin/ExamplesTest.kt

```diff
@@ -93,30 +92,6 @@ class ExamplesTest : StringSpec() {
             }
         }
 
-        "notice-pre-processor.kts can be compiled" {
```

Comment:
Shouldn't the removal of `examples/notice-pre-processor.kts` rather be part of this commit instead of the previous commit?

Reply:
Done.